nimsuggest/nimsuggest.nim

```diff
@@ -487,7 +487,9 @@ proc mainThread(graph: ModuleGraph) =
 var
   inputThread: Thread[ThreadParams]
 
-proc mainCommand(graph: ModuleGraph) =
+type ProgNimsuggest=ref object of ProgBase
```

Comment:
Empty subclass means you don't need any inheritance/OOP here.

BTW Spaces around the '='. Just like you saw it *everywhere* else in Nim's whole codebase.

Reply:
done